server/historian/package.json

```diff
@@ -28,9 +28,8 @@
 		"start": "node packages/historian/dist/www.js",
 		"start:debug": "docker-compose -f docker-compose.yml -f docker-compose.dev.yml -f docker-compose.debug.yml up",
 		"start:dev": "docker-compose -f docker-compose.yml -f docker-compose.dev.yml up",
-		"test": "lerna run test --stream --no-bail --no-sort",
-		"tsc": "tsc",
-		"tslint": "tslint 'src/**/*.ts'"
+		"test": "pnpm run -r --stream --no-bail --no-sort test",
```

Comment:
One thing this will be different is that I believe pnpm default to only use 4 cores, and lerna use the number core of the machine.   Did we do a comparison of time?

Reply:
might need to add `--workspace-concurrency=0`

Reply:
Set it in the .npmrc for all release groups.